packages/itwinui-react/src/core/Select/Select.tsx

```diff
@@ -310,6 +311,7 @@ export const Select = <T,>(props: SelectProps<T>): JSX.Element => {
         key: `${label}-${index}`,
         isSelected,
         startIcon: startIcon,
+        endIcon: isSelected ? <SvgCheckmark aria-hidden /> : null,
```

Comment:
Unit tests for this?

Reply:
Ahh idk, comparing svg nodes within jest always feels pointless. We do have visual tests, so I figured it would be enough